lightbluetent/templates/home/directory_base.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
```

Comment:
ideally we should avoid duplicating base templates, especially in the case of a unique page such as the directory, but this is OK for now